packages/ui/src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { AnalyticsContextProvider } from '@prezly/analytics-nextjs';
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { COMPANY_INFORMATION } from '../Boilerplate/__mocks__';
+
+import { Footer } from './Footer';
+
+export default {
+    title: 'Components/Footer',
+    component: Footer,
+    parameters: {
+        design: {
+            type: 'figma',
+            url: '',
+        },
+        layout: 'fullscreen',
+    },
+    decorators: [
+        (Story) => (
+            <AnalyticsContextProvider>
+                <Story />
+            </AnalyticsContextProvider>
+        ),
+    ],
+} as Meta<typeof Footer>;
+
+const FooterTemplate: StoryFn<typeof Footer> = (args) => <Footer {...args} />;
+
+export const Default = FooterTemplate.bind({});
+Default.args = {
```

Comment:
As with previous PRs, can we get more story variations for all components? We need to make sure that the component looks good when not every field is populated.

Reply:
Hey @fgyimah, I asked to add variations on all components 🤔 
Both ContactCard and Boilerplate can look completely different without particular fields